ui/article-aem-renderer/src/main/java/org/cru/godtools/article/aem/activity/AemArticleActivity.kt

```diff
@@ -43,6 +45,9 @@ class AemArticleActivity : BaseSingleToolActivity(false, false) {
     private lateinit var syncTask: ListenableFuture<Boolean>
     private var article: Article? = null
 
+    private var pendingAnalyticsEvent = false
+    private var hasScreenEventTriggered: Boolean = false
```

Comment:
the small nit that is actually my fault (and I haven't gone to fix yet) is that this field is no longer used and can be deleted.